Main.py

```diff
@@ -287,22 +287,22 @@ def on_prepare(event):
 
             # Make sure factory image is selected
             if not self._config.firmware_model:
-                print("ERROR: Select a Valid Factory Image.")
+                print("ERROR: Select a valid factory image.")
                 return
 
             # Make sure platform-tools is set and adb.exe and fastboot.exe are found
             if not self._config.platform_tools_path:
-                print("ERROR: Select Android Platform Tools (adb)")
+                print("ERROR: Select Android platform tools (ADB)")
```

Comment:
Part widget, part not.
I think This would be good like this
"ERROR: Select Android Platform Tools (ADB)"